### PR TITLE
Adds config file for CircleCI so that deployments can happen automatically

### PR DESCRIPTION
**Pull request by trezy**

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+machine:
+  node:
+    version: 4.0.0
+
+deployment:
+  production:
+    branch: master
+    commands:
+      - npm run deploy
```
Comment (rylnd) [inline circle.yml]: I don't think this is a script that `npm` knows about. As per the README, we `$(npm bin)/gulp deploy` to deploy.

Comment (trezy) [inline circle.yml]: Good call. Fixed that in a separate PR. :-)

Comment (rylnd) [inline circle.yml]: :+1:

**Comment by rylnd** [comment]
hey look, an old branch

**Pull request merged**